testing/trino-tests/src/test/java/io/trino/tests/TestDistributedEngineOnlyQueries.java

```diff
@@ -137,6 +138,14 @@ public void testRowTypeWithReservedKeyword()
                 "VALUES 1");
     }
 
+    @Test
+    public void testExplain()
+    {
+        assertExplain(
+                "explain select name from nation where abs(nationkey) = 22",
```

Comment:
uppercase keywords: EXPLAIN SELECT ..

also, as reported in https://github.com/trinodb/trino/issues/6655, can date be any special?